killbill-integration-tests/mixin-utils/helper.rb

```diff
@@ -65,6 +65,11 @@ def add_catalog_simple_plan(plan_id, product_name, product_category, currency, a
       KillBillClient::Model::Catalog.add_tenant_catalog_simple_plan(simple_plan, user, 'Test', 'Upload simple plan', options)
     end
 
+    def upload_tenant_user_key_value(key, value, user, options)
+      KillBillClient::Model::Tenant.upload_tenant_user_key_value(key,value, user, 'New Config', 'Upload config for tenant', options)
+      sleep(TIMEOUT_SEC)
```

Comment:
Why is the timeout needed? I would expect the call to be synchronous?

Reply:
I stumble that sometimes, if the test is focused in testing detail mode data and I set it as `detail_mode` and a the previous is `aggregated_mode`, it takes the previous setting and not the current mode, thus fail the test. Maybe is my environment, but that is the reason.